File: Milliways/Models/MenuItem.swift
//
//  MenuItem.swift
//  Milliways
//
//  Created by gilm on 05/11/2025.
//

import SwiftUI

struct MenuItem: Identifiable {
    let id = UUID()
    let name: String
    let description: String
    let price: Double
    let color: Color
    var imageName: String? = nil
}

File: android/app/src/main/java/com/mobilenext/Milliways/model/MenuItem.kt
package com.mobilenext.Milliways.model

import androidx.annotation.DrawableRes
import androidx.compose.ui.graphics.Color
import java.util.UUID

data class MenuItem(
    val id: String = UUID.randomUUID().toString(),
    val name: String,
    val description: String,
    val price: Double,
    val color: Color,
    @DrawableRes val imageRes: Int? = null
)

fun formatPrice(value: Double): String = "₭%.2f".format(value)
